course/resources/eeb430-syllabus.md

```diff
@@ -0,0 +1,124 @@
+# EEB430H1 Theoretical Ecology [24L, 12P]
+Theoretical ecology uses models to explain biological phenomena such as the maintenance of biodiversity, population growth, competition, eco-evolutionary dynamics, epidemiology, spatial ecology, and species extinction. This course is designed to meet a growing demand for reproducible, openly accessible, analytically thorough, and well documented science. Students will learn to develop ecological population models, analyze data and document their research using the R programming language. No prerequisite programming experience is required.
+
+Prerequisites: MAT 136, BIO220H1, EEB225H1, and two of EEB319H1, EEB321H1,
+EEB322H1, EEB323H1
+
+# Time and location
+Tue and Thu 2-4 pm.
+
+Tue, Ramsay Wright (RW 109)
+South west corner of Harbord and St George, 1st floor, enter from St George street.
+
+Thu, Carr Hall (SMC 325)
+Corner of Queen's Park East and St. Joseph Street, 3rd floor, St. Michael's College.
+
+## Computer labs
+Students will learn theoretical ecology concepts and how to program in R. Both lecture halls have access to individual computers for the students. To use the computer workstations, students can login with their UTORid and password. Installed programs and packages are wiped from the computers daily. Saved files will be accessible from any computer via the student's login. Students can use any of the lecture halls when there are no classes scheduled. See the [online schedules](http://lab.chass.utoronto.ca/carr.php) for available times.
+
+# Contacts
+
+## Instructor:
+Prof. Martin Krkosek, martin.krkosek@utoronto.ca, 416-978-3839, RW402
```

Comment:
I think he should be listed, there needs to be someone they can go to for EEB related things or if they have complaints about us (hope they don't, but it is the reason for having at least one prof related to a course).